src/base.iced

```diff
@@ -233,23 +235,25 @@ class Verifier
     err = null
     if not /^[\x20-\x7e]+$/.test json_str_utf8_trimmed
       err = new Error "All JSON proof characters must be in the visible ASCII set (properly escaped UTF8 is permissible)"
-    else
-      [e, @json] = katch (() -> JSON.parse json_str_buf)
-      err = new Error "Couldn't parse JSON signed message: #{e.message}" if e?
-      if not err?
-        if @strict and ((ours = trim(json_stringify_sorted(@json))) isnt json_str_utf8_trimmed)
-          err = new Error "non-canonical JSON found in strict mode (#{errsan ours} v #{errsan json_str_utf8_trimmed})"
-        else
-          await @base._v_check {@json}, defer err
-    cb err, @json, json_str_utf8
+      return cb err
+    [e, json_tmp] = katch (() -> JSON.parse json_str_buf)
+    if e?
+      err = new Error "Couldn't parse JSON signed message: #{e.message}"
+      return cb err
+    if @strict and ((ours = trim(json_stringify_sorted(@json))) isnt json_str_utf8_trimmed)
```

Comment:
`@json` does not look set here anywhere in this function like it used to be on line 237. is this expected?

Reply:
maybe we need
```
if @strict and ((ours = trim(json_stringify_sorted(json_tmp))) isnt json_str_utf8_trimmed)
```
here

Reply:
https://github.com/keybase/proofs/pull/170